### PR TITLE
UI changes on Source page for optional surveys.

### PR DESCRIPTION
**Pull request by cassidysymons**

### CODE REVIEW

File: microsetta_interface/implementation.py [hunk 1]
```diff
@@ -106,8 +106,16 @@ class Source:
     5: 'Questions about your interest in the microbiome',
     6: 'Questions specific to COVID19',
     VIOSCREEN_ID: 'Our standard food frequency questionnaire',
-    MYFOODREPO_ID: 'The MyFoodRepo picture-based, artificial '
-                   'intelligence backed, diet assessment platform'
+    MYFOODREPO_ID: 'US Participants only: By sharing photos of the food you '
```
Comment (wasade) [inline microsetta_interface/implementation.py:109]: The instrument is fit for US participants but it is not limited to them

File: microsetta_interface/implementation.py [hunk 2]
```diff
@@ -1090,6 +1098,11 @@ def get_source(*, account_id=None, source_id=None):
     per_source_not_taken = [translate_survey_template(s)
                             for s in per_source_not_taken]
 
+    # quick hack to move MyFoodRepo to top of list, if available
+    per_source_not_taken = sorted(per_source_not_taken,
+                                  key=lambda s: s['survey_template_id'],
+                                  reverse=True)
```
Comment (wasade) [inline microsetta_interface/implementation.py:1104]:
```suggestion
    per_source_not_taken = per_source_not_taken[::-1]
```

As an alternative, ill defer to you which you'd prefer

File: microsetta_interface/templates/source.jinja2 [hunk 3]
```diff
@@ -245,72 +245,81 @@
...
+    </div>
+    <br />
+    {% endif %}
+
+    {% if surveys|length > 0 %}
+    <hr style="border-top: dotted 1px"/>
+    <p>{{ _('Completed surveys') }}:</p>
+    <div class="list-group">
+        {% for survey in surveys %}
+        <a>
```
Comment (wasade) [inline microsetta_interface/templates/source.jinja2:296]: Why wrap this in `a`?

**Review by wasade — APPROVED** [review]
Minor items

File: microsetta_interface/implementation.py (hunk 1 above)
Comment (cassidysymons) [reply microsetta_interface/implementation.py:109]: Are we trying to discourage international participants from using MFR, but allowing them to if they so desire? Or do we welcome anyone and everyone to use it?

**Comment by wasade** [comment]
Best for Se Jin to comment, unsure what her gh handle is tho

File: microsetta_interface/templates/source.jinja2 (hunk 3 above)
Comment (cassidysymons) [reply microsetta_interface/templates/source.jinja2:296]: Honestly, no idea, I just moved the entire block for completed surveys and changed "Surveys taken" to "Completed surveys." Scrolling back through the history of the file, it appears the tag has been there for years. Want me to remove it?

Comment (wasade) [reply microsetta_interface/templates/source.jinja2:296]: I think it would be correct to remove as we don't have my hyperlinks in that block, thanks!

**Comment by wasade** [comment]
+1 thanks!!

**Pull request merged**